### DISCUSSION TITLE
region 'update' event problem

### DISCUSSION THREAD

**Discussion by martvin-github** [katspaugh/wavesurfer.js#3255] [Q&A]
I am trying to find the option I used in version 6, namely wsRegion.on('update'), which gave me the start and end position of the region in real time, but despite all my efforts I am not succeeding. I'm not a good javascript programmer, rather an amateur, perhaps I'm making a mistake. But does this function work in version 7? It is mentioned in the manual, but perhaps my use is incorrect or I do not understand something. 
Is there an example of its use? 
I was using this function to prevent overlap.
Sorry for my poor English

**Comment by katspaugh** [comment]
Each region has an [`update` event](https://wavesurfer.xyz/docs/types/plugins_regions.RegionEvents), and the Regions plugin instance also has a [`region-updated` event](https://wavesurfer.xyz/docs/types/plugins_regions.RegionsPluginEvents).
You can see an example of the latter here: https://wavesurfer.xyz/examples/?regions.js on line 65.
